scripts/defaults

```diff
@@ -3,7 +3,7 @@ echo "this is the file defaults"
 export PYTHON=${PYTHON:-"python"}
 export VIRTUAL_ENV=${VIRTUAL_ENV:-"$PWD/socorro-virtualenv"}
 # Valid values for BUILD_TYPE are "rpm" (which we use in socorro-infra) and
-# "ci" # which we use in Travis CI.
+# "ci" # which we use in CI.
```

Comment:
I wrote this after spending *quality time* with these scripts. I'm pretty sure you're right in that this leaves us with only one build type in use. We could remove it, but it'd be worth verifying the changes.

As I recall, I ended up setting up a Vagrant environment (which I don't think I can reproduce now without effort) to test things. It wasn't fun.

All these scripts go away after we switch to the new infra, so I'd just leave it as is rather than try to figure out how to test everything.